.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,41 @@
+name: Linux tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Run unit test
+      run: |
+        export PATH=$PATH:$HOME/.local/bin
+        make verify
+
+    - name: Run build test
+      run: |
+        export PATH=$PATH:$HOME/.local/bin
+        make
+
+    - name: Make container test
+      run: |
+        export PATH=$PATH:$HOME/.local/bin
+        make container
```

Comment:
@andyzhangx we have this test in travis CI as well as github Action. This test seems to be failing in travis due to permission issues. Shall we remove travis? It seems redundant.

Reply:
point is travis originally works, I still want to keep travis test running for a period of time, thanks.
And also now github action failed, need to fix that first.